src/mobile/metro.config.js

```diff
@@ -0,0 +1,9 @@
+const { makeMetroConfig } = require('@rnx-kit/metro-config');
+const MetroSymlinksResolver = require('@rnx-kit/metro-resolver-symlinks');
+
+module.exports = makeMetroConfig({
+  projectRoot: __dirname,
+  resolver: {
+    resolveRequest: MetroSymlinksResolver(),
+  },
+});
```

Comment:
Wondering what is this file for?
Does expo generate that file?

Reply:
Yes it's auto generated